core/templates/pages/story-editor-page/services/story-editor-state.service.ts

```diff
@@ -282,6 +283,14 @@ export class StoryEditorStateService {
     return this._storyIsBeingSaved;
   }
 
+  setCurrentNodeAsPublishable(val: boolean): void {
```

Comment:
What is "val"? Please use a more descriptive name.

Reply:
Updated.